builds/android/app/src/main/java/org/easyrpg/player/player/EasyRpgPlayerActivity.java

```diff
@@ -83,6 +88,7 @@ protected void onCreate(Bundle savedInstanceState) {
 				getWindow().setFlags(0x01000000, 0x01000000);
 			}
 		} catch (Exception e) {
+            e.printStackTrace();
```

Comment:
but you should have configured your editor to use Tabs or at least "keep source file identation" >.>